src/daemon/ca/publishing.rs

```diff
@@ -296,6 +297,17 @@ impl CaObjects {
         res
     }
 
+    /// Returns all PublishElements in all repositories (if there is more than one)
+    pub fn all_publish_elements(&self) -> Vec<PublishElement> {
+        let mut all_elements = vec![];
+
+        for (_, mut elements) in self.repo_elements_map().into_iter() {
```

Comment:
Could you append all .values() from the hashmap and then flatten here instead of this loop construction?

Reply:
pushed "bffb078fa56b072b0813ab36948743a1e2b20b47"